CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
     rely on this method, please, replace it with `java.net.URL#URL(String)`
     constructor call
 
+*   Deprecated Basket getters/setters: `note`, `amountTotalVat`, `amountTotalGross`,
```

Comment:
'note' is still a valid field and shall not be deprecated.
https://unz.atlassian.net/wiki/spaces/UCC/pages/825820185/UMC+-+Tech+Spec+-+Basket+API+v2+for+SDKs+PHP+SDK+Java+SDK#v2-Basket

Reply:
"amountTotal" on the other hand needs to be deprecated

Reply:
Needs to be decided on by Rene